Makefile

```diff
@@ -287,6 +292,19 @@ $(CSS_DEST_DIR)/%.css: $(CSS_SRC_DIR)/%.css
 	$(call printfile,$<) >> $@
 
 # maintenance
+ifdef WINDOWS
+clean:
+	$(call RM, $(VERSION_CONFIG))
+	$(call RM, $(DEFAULTS_CONFIG))
+	$(call RM, $(DOCS_PAGE_LIST))
+	$(call RM, $(DOCS_VERSION_DATA))
+	$(call RM, $(TOC_FILES))
+	$(call RM, $(PLUGINS_APP))
+	$(call RM, $(FETCHED_FILES))
+	-$(call RMDIR, $(CSS_DEST_DIR))
+	-$(call RMDIR, $(PROD_DIR))
+	-$(call RMDIR, $(DEV_DIR))
+else
```

Comment:
If you have `RM` and `RMDIR` defined for all platforms, you don't have to add Windows-specific logic here.